packages/cyberstorm/src/components/Button.tsx

```diff
@@ -27,14 +27,34 @@ export const Button: React.FC<ButtonProps> = (props) => {
       type="button"
       /* TS is not aware of defaultProps of function components. */
       /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
-      className={`${styles.button} ${styles[buttonStyle!]}`}
+      className={`${styles.root} ${getStyle(buttonStyle)}`}
       onClick={onClick}
     >
       {leftIcon}
-      {label ? <div className={styles.buttonLabel}>{label}</div> : null}
+      {label ? <div className={styles.label}>{label}</div> : null}
       {rightIcon}
     </button>
   );
 };
 
-Button.defaultProps = { buttonStyle: "button__default" };
+Button.defaultProps = { buttonStyle: "default" };
+
+function getStyle(style: string) {
```

Comment:
* This could be typed as `style: ButtonProps["buttonStyle"]`. In practice it probably doesn't matter much, but at least it would catch typos if someone would write the value manually within the component.
* Except if we're calling it `colorScheme`, the naming is bit off
* Also using a switch seems a bit verbose. Would something like this be better?

```
const getStyle = (scheme: ButtonProps["colorScheme"]) => {
  return {
    danger: styles.button__danger,
    default: styles.button__default,
    defaultDark: styles.button__defaultDark,
    defaultWithBorder: styles.button__defaultWithBorder,
    primary: styles.button__primary,
    specialGreen: styles.button__specialGreen,
    specialPurple: styles.button__specialPurple,
  }[scheme];
};
```

TS would now complain if any of the color schemes defined in component props doesn't have a style pair defined here.

Reply:
Very nice, more compact and all👍